Mathlib/NumberTheory/ModularForms/ArithmeticSubgroups.lean

```diff
@@ -6,6 +6,10 @@ Authors: David Loeffler
 import Mathlib.Data.Real.Basic
 import Mathlib.GroupTheory.Commensurable
 import Mathlib.LinearAlgebra.Matrix.GeneralLinearGroup.Defs
+import Mathlib.Topology.Algebra.IsUniformGroup.Basic
+import Mathlib.Topology.Algebra.Ring.Real
+import Mathlib.Topology.Instances.Matrix
+import Mathlib.Topology.MetricSpace.Isometry
```

Comment:
This seems like a lot of extra imports. Could this be a new file?

Reply:
If you look at the import-summary-bot output, it shows that almost everything which imports `ArithmeticSubgroups` already imports these files anyway, so the change in the import tree is very localized (there is just 1 downstream file, `CongruenceSubgroups`, which gets a nontrivial import increase as a result). Also, I created `ArithmeticSubgroups.lean` myself only a few weeks ago, and it was always the plan that it would contain a proof that arithmetic subgroups are discrete.